tlsutil/config.go

```diff
@@ -319,29 +318,30 @@ func (c *Configurator) UpdateAutoTLS(manualCAPems, connectCAPems []string, pub,
 	c.autoTLS.cert = &cert
 	c.caPool = pool
 	c.autoTLS.verifyServerHostname = verifyServerHostname
-	c.version++
+	atomic.AddUint64(&c.version, 1)
+	c.log("UpdateAutoTLS")
 	return nil
 }
 
 func (c *Configurator) UpdateAreaPeerDatacenterUseTLS(peerDatacenter string, useTLS bool) {
-	c.Lock()
-	defer c.Unlock()
-	c.version++
```

Comment:
Contextually I'm guessing you removed this because it probably wasn't supposed to be here in the first place.

Reply:
Oh, hmm, no this might have been a mistake. I had tried some other refactors before restoring the `version` increments to the Update functions. I think I should restore it here.

The version is only used in trace logs, so not super critical, but I guess since this is an update function it would still be good to increment the version.